src/pages/provider/ProviderCapacityTable.tsx

```diff
@@ -178,9 +191,9 @@ export const ProviderCapacityTable: React.FC<ProviderCapacityTableProps> = ({
         {!capacities ? (
           <Skeleton width={200} height={34} count={1} />
         ) : (
-          capacities.total !== null && (
+          capacities.length > limit && (
             <Pagination
-              pages={getTotalPages(capacities.total)}
```

Comment:
The `capacities.total` is equal count all CCs so I consider fetch all records here right now